app/models/concerns/account_avatar.rb

```diff
@@ -34,7 +34,7 @@ def avatar_remote_url=(url)
 
       self.avatar              = URI.parse(parsed_url.to_s)
       self[:avatar_remote_url] = url
-    rescue OpenURI::HTTPError => e
+    rescue OpenURI::HTTPError, OpenSSL::SSL::SSLError, Paperclip::Errors::NotIdentifiedByImageMagickError => e
```

Comment:
I'd put those into separate commits.

I'll approve this PR, can you can decide to split it up or not.